.changes/next-release/Feature-97c6535a-a446-4f56-beaa-da5ee8d4a5f9.json

```diff
@@ -0,0 +1,4 @@
+{
+    "type": "Feature",
+    "description": "Adding CloudWatch Logs functionality"
```

Comment:
either past tense or simply a noun ("CloudWatch Logs UI") fits better in a list of features, based on the wording in our previous changelogs.

present participle ("fooing") might work for a live stream or a tweet of something being done at this very moment ("I am doing foo").